packages/waas/src/base.ts

```diff
@@ -80,7 +82,9 @@ export class SequenceWaaSBase {
 
   constructor(
     public readonly config = { network: 1 } as SequenceBaseConfig,
-    private readonly store: Store = new LocalStore()
+    private readonly store: Store = new LocalStore(),
+    private readonly cryptoBackend: SubtleCryptoBackend | null = getDefaultSubtleCryptoBackend(),
+    private readonly secureStoreBackend: SecureStoreBackend
```

Comment:
I wonder if we should include a default for `secureStoreBackend` too..?

Reply:
Since `secureStoreBackend` is required, I added a default in `auth.ts` and made sure it was null-checked before coming here.